blueprints/addon/files/.npmrc

```diff
@@ -0,0 +1,8 @@
+# these settings allow us to be good stewards of the ecosystem
+# end-consumers can flip these settings in their own projects,
+# but library authors should not allow for less-than-strictest
+# behavior.
+#
+# For more information: https://pnpm.io/npmrc
+auto-install-peers=false
```

Comment:
I thought, from the RFC discussion, that auto-install-peers had become a default

Reply:
Yes, as mentioned in the RFC, it's not good for library authors, because we must declare peers and deps correctly so that consumers _can_ rely on auto-install peers behaving correctly.

The auto install behavior does not propagate through the graph layers.